packages/base/src/signal-manager.ts

```diff
@@ -3,6 +3,7 @@ import { EventEmitter } from 'events';
 export declare interface SignalManager {
 
     fireTooltipSignal(tooltip: { [key: string]: string }): void;
+    fireEventClickSignal(payload: {[timeStamp: string]: number }): void;
```

Comment:
"timestamp" should be a different name. Timestamp is one of the keys. Later on we might have multiple once.
I wonder if we should pass strings instead of numbers and the signal receivers have to convert them if necessary.
Name could be fireSelectionChangedSignal(), selection changed (see comment below on signal name)